tools/fix_go_deps.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
can you add an option to ./buildfix.sh that does this so we continue to have one script that will "ready" your build for review?

Reply:
Done, added a `--fix-go-deps` flag for this. Also added an `--all` flag that is the same as `--gazelle --fix-go-deps`.